CHANGELOG.md

```diff
@@ -14,11 +14,14 @@ Starting with the next release, we would like to follow
 https://keepachangelog.com/en/1.1.0/. Please add to relevant
 subsections here on every PR where this is applicable.
 
+### Added
+
+* Add support to skip staging and/or caching files to the local filesystem while uploading files in artifacts
```

Comment:
thanks for adding this! up to you but we usually also add the github user handle and the PR. Something, like Tim did bellow, so in the end of your comment just add:
`by @ibindlish in https://github.com/wandb/wandb/pull/7098`

Reply:
Sure, will do!